src/style.css

```
@@ -0,0 +1,132 @@
+/* stylelint-disable font-family-no-missing-generic-family-keyword */
+:root {
+  ---blue: #00acdc;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  max-width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+}
+
+.card__image {
+  height: 134px;
+}
+
+.card__title {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+  margin: 0;
+  margin-top: 40px;
+}
+
+.card__describe {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+}
+
+.card__describe-code {
+  font-family: "Roboto";
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  margin-top: 4px;
+}
+
+.card__describe-reviewer {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-top: 16px;
+}
+
+.card__describe-reviewer_count {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #060b35;
+}
+
+.stars {
+  display: flex;
+  gap: 4px;
+}
+
+.stars__star {
+  background-image: url("./images/star.svg");
+  background-repeat: no-repeat;
```

forgot to writh in prev review 
```suggestion
  background-repeat: no-repeat;
  backround-position: center;
```